app/src/main/res/layout/opoc_filesystem_fragment.xml

```diff
@@ -39,8 +39,12 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:minHeight="100dp"
+            app:fastScrollEnabled="true"
+            app:fastScrollHorizontalThumbDrawable="@drawable/scrollbar_thumb"
+            app:fastScrollHorizontalTrackDrawable="@drawable/scrollbar_track"
```

Comment:
I don't any horizontal scrolling is needed? Especially also on the main screen there is horizontal swiping which contradicts horizontal scrolling in subviews.

Reply:
I noticed that if we added the `app:fastScrollEnabled="true"`, then the following 4 related Drawable attributes are all indispensable, even the horizontal scrollbar will not be revealed forever,
`app:fastScrollHorizontalThumbDrawable`;
`app:fastScrollHorizontalTrackDrawable`;
`app:fastScrollVerticalThumbDrawable`;
`app:fastScrollVerticalTrackDrawable`,
otherwise this APP will crash when launched.

Reply:
So we cannot delete `app:fastScrollHorizontalThumbDrawable` and `app:fastScrollHorizontalTrackDrawable`.

Reply:
@gsantner , I just tried it, there is no contradicts in horizontal swiping and horizontal scrolling in subviews.